packages/fxa-settings/src/components/Settings/FlowRecoveryKeyInfo/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { FlowContainer } from '../FlowContainer';
+import { ProgressBar } from '../ProgressBar';
+import { SecurityShieldImage } from '../../images';
+import { ShieldIconListItem, KeyIconListItem } from '../../IconListItem';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { useFtlMsgResolver } from '../../../models';
+import { logViewEvent, usePageViewEvent } from '../../../lib/metrics';
+
+export type FlowRecoveryKeyInfoProps = {
+  navigateForward: () => void;
+  navigateBackward: () => void;
+};
+
+export const viewName = 'settings.account-recovery.info';
+export const backwardNavigationEventName = 'navigate-forward';
+export const forwardNavigationEventName = 'navigate-backward';
+
+export const FlowRecoveryKeyInfo = ({
+  navigateForward,
+  navigateBackward,
+}: FlowRecoveryKeyInfoProps) => {
+  usePageViewEvent(viewName);
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const localizedContainerPageTitle = ftlMsgResolver.getMsg(
+    'flow-recovery-key-info-page-title',
+    'Account Recovery Key'
+  );
+
+  return (
+    <FlowContainer
+      title={localizedContainerPageTitle}
+      customBackButtonTitle="Settings"
+      customBackButtonLocalizationId="flow-recovery-key-info-back-button-title"
+      onBackButtonClick={() => {
+        logViewEvent(viewName, backwardNavigationEventName);
```

Comment:
Q: Should this also log the current step? Or is the `viewName` sufficiently unique to know which step we're on?

```js
export const viewName = 'settings.account-recovery.info';
export const backwardNavigationEventName = 'navigate-forward';
export const forwardNavigationEventName = 'navigate-backward';
```

Reply:
I think it's worth logging on click because (in theory) someone might initiate the action and have it fail, but I'm open to removing it! We haven't really nailed down what we are/aren't tracking, which I guess will get hammered out in FXA-7249?

Reply:
Lets' keep for now and review in [FXA-7249](https://mozilla-hub.atlassian.net/browse/FXA-7249)!